packages/doenetml-worker-rust/lib-rust-macros/src/component_state_methods.rs

```diff
@@ -606,3 +599,6 @@ pub fn add_dependency_data_impl(_attr: TokenStream, item: TokenStream) -> TokenS
         _ => panic!("`add_standard_component_fields` has to be used with structs."),
     }
 }
+
+#[cfg(test)]
```

Comment:
Is this doing something other than reminding us to write tests?

Reply:
No, just a reminder. I almost wrote some tests, but then got distracted...